lib/shared/wellbeing_graph.dart

```diff
@@ -14,9 +14,10 @@ const RECOMMENDED_STEPS_IN_WEEK = 70000;
 /// key to retreive [bool] from [SharedPreferences] that is true if the tutorial has been completed
 const WB_TUTORIAL_DONE_KEY = "wb_tutorial_done";
 
+
 /// a [StatefulWidget] that displays the last wellbeing items in a graph,
 /// along with (optionally) a share button, and (optionally) a tutorial button
-///
+
 /// REVIEW: maybe switch to this if have time: https://pub.dev/packages/fl_chart
```

Comment:
I think this needs to be connected for it to be considered a doc comment for WellbeingGraph widget. e.g. hover over WellbeingGraph in your editor and see if the entire comment is displayed, not just the REVIEW part.

Reply:
![image](https://user-images.githubusercontent.com/55795994/110322122-8be4fb80-800a-11eb-8142-93c8759eeb27.png)

Reply:
I'll connect it so it looks nicer